Makefile

```
@@ -50,12 +50,13 @@ SHELL ?= bash
 -include hack/make/tests/*.mk
 -include hack/make/deploy/*.mk
 -include hack/make/helm/*.mk
+-include hack/make/rest-client/*.mk
 
 ## Installs dependencies
 deps: prerequisites/setup-pre-commit prerequisites/kustomize prerequisites/controller-gen
 
 ## Builds the operator image and pushes it to quay with a snapshot tag
-build: images/push/tagged
+build: images/push/tagged swagger/generate-rest-client
```

Are those in the correct order? I imagine the rest client must be generated before the image is pushed.
also:
```suggestion
build: images/push/tagged swagger/generate
```